src/middleware/srgssr.js

```diff
@@ -471,6 +472,8 @@ class SrgSsr {
    * @returns {Promise<import('./typedef').ComposedSrcMediaData>} - The composed source media data.
    */
   static async getSrcMediaObj(player, srcObj) {
+    player.trigger('pillarbox-monitoring/sessionstart');
```

Comment:
The name of this event describes the action that will be taken by the listener instead of the source of the event. I would suggest changing the name of the event to something like: `pillarbox/start-source-setup` or directly calling the function that we want to trigger in the monitoring object.

Reply:
Make sens 👍🏽